specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -3278,6 +3284,10 @@
           "title": "Whether each node is allocated its own public IP.",
           "description": "Some scenarios may require nodes in a node pool to receive their own dedicated public IP addresses. A common scenario is for gaming workloads, where a console needs to make a direct connection to a cloud virtual machine to minimize hops. For more information see [assigning a public IP per node](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools). The default is false."
         },
+        "enableCustomCATrust": {
+          "type": "boolean",
+          "description": "Whether to enable Custom CA Trust feature."
```

Comment:
I think this at least needs more details. What is this feature? When might you want to use it? Do we have links to a detailed article or documentation we could include here? If not, we should explain what this feature is and when you might want to use it. You can change this bit to the `title` and add a `description` with more details.

Reply:
Also document the default.

Reply:
```suggestion
          "description": "Whether to enable Custom CA Trust feature. When set to true, AKS deploys a daemonset and host services to sync custom certificate authorities from a user-provided config map intto node trust stores."
```